specs/agents/tracing-spans-service-target.md

```diff
@@ -0,0 +1,226 @@
+## Span service target
```

Comment:
[for reviewer] this spec currently describes how to transition from current to new fields, once the APM Server implementation will be complete (Phase 3), we can simplify the spec to explain which fields are required depending on the APM server version.